openedx/features/job_board/models.py

```diff
@@ -15,17 +17,31 @@ class Job(TimeStampedModel):
     """
     title = models.CharField(max_length=255)
     company = models.CharField(max_length=255)
-    type = models.CharField(max_length=255, choices=JOB_TYPE_CHOICES)
-    compensation = models.CharField(max_length=255, choices=JOB_COMPENSATION_CHOICES)
-    hours = models.CharField(max_length=255, choices=JOB_HOURS_CHOICES)
+    type = models.CharField(max_length=255, choices=JOB_TYPE_CHOICES, blank=False,
+                            default=0, help_text='Please select whether the job is onsite or can be done remotely.')
+    compensation = models.CharField(max_length=255, choices=JOB_COMPENSATION_CHOICES,
+                                    blank=False, default=0, help_text='Please select the type of compensation you are '
+                                                                      'offering for this job.')
+    hours = models.CharField(max_length=255, choices=JOB_HOURS_CHOICES,
+                             blank=False, default=0, help_text='Please select the expected number of working hours '
+                                                               'required for this job.')
     city = models.CharField(max_length=255)
     country = CountryField()
-    description = models.TextField()
-    function = models.TextField(blank=True, null=True)
-    responsibilities = models.TextField(blank=True, null=True)
+    description = models.TextField(help_text='Please share a brief description of the job.')
+    function = models.TextField(blank=True, null=True, help_text='Please share details about the expected functions '
+                                                                 'associated with the job.')
+    responsibilities = models.TextField(blank=True, null=True, help_text='Please share the responsibilities '
+                                                                         'associated with the job.')
     website_link = models.URLField(max_length=255, blank=True, null=True)
-    contact_email = models.EmailField(max_length=255)
-    logo = models.ImageField(upload_to='job-board/uploaded-logos/', blank=True, null=True)
+    application_link = models.URLField(max_length=255, blank=True, null=True,
+                                       help_text='Please share a link to the job application')
+    contact_email = models.EmailField(max_length=255, help_text='Please share a contact email for this job.')
+    logo = models.ImageField(upload_to='job-board/uploaded-logos/', blank=True, null=True,
+                             validators=[
+                                 FileExtensionValidator(LOGO_ALLOWED_EXTENSION),
+                                 validate_file_size
+                             ],
+                             help_text='Please upload a file with your company\'s logo. (maximum 2MB)')
```

Comment:
Should we also add the suggested aspect ratio in this help text?

Reply:
as discussed, we will proceed with this after getting a go ahead from product team.